runtime/stdlib/contracts/test.cdc

```diff
@@ -122,10 +219,10 @@ pub contract Test {
             return self.backend.events(type)
         }
 
-        /// Resets the state of the blockchain.
+        /// Resets the state of the blockchain at the given height.
         ///
-        pub fun reset() {
-            self.backend.reset()
+        pub fun reset(_ height: UInt64) {
```

Comment:
Embrace the use of argument labels, they help writers and readers of the code. For example here, consider a short but useful argument label, e.g. `to` to indicate the number is a block height, or a more verbose / descriptive `toHeight`
```suggestion
        pub fun reset(to height: UInt64) {
```

Same below for the other `reset` function

Reply:
Nice, I wasn't aware of this little gem :100: 
Updated in https://github.com/onflow/cadence/pull/2696/commits/aee08f0d524c5b237ccb052618abfa9d3e507ab7